tests/phpunit/tests/theme.php

```diff
@@ -965,6 +965,50 @@ public function test_block_theme_should_load_separate_core_block_assets_by_defau
 		);
 	}
 
+	/**
+	 * Tests that block themes load block assets on demand by default.
+	 *
+	 * @ticket 61965
+	 *
+	 * @covers ::_add_default_theme_supports
+	 * @covers ::wp_should_load_block_assets_on_demand
+	 */
+	public function test_block_theme_should_load_block_assets_on_demand_by_default() {
+		$this->helper_requires_block_theme();
+
+		add_filter( 'should_load_block_assets_on_demand', '__return_false' );
+
+		$this->assertFalse(
+			wp_should_load_block_assets_on_demand(),
+			'Could not disable loading block assets on demand.'
```

Comment:
I'm probably missing something. But why does the comment here say that it wasn't possible to disable loading block assets on demand?

From what I read from the code here we are setting it to false and then checking that the value is false which works as expected? 🤔

Reply:
The message is for the case this is failing, in which case the filter to set this to `false` would (for some reason) not work as expected.

Reply:
Ahhh thanks :)